src/core/server/root/index.ts

```diff
@@ -84,7 +79,9 @@ export class Root {
       this.log.fatal(reason);
     }
 
-    await this.server.stop();
+    if (this.server !== undefined) {
```

Comment:
__question:__ hmm, when `this.server` can be `undefined`? It seems to be marked as `readonly`.